lib/shopify_cli/theme/theme.rb

```diff
@@ -176,8 +176,15 @@ def live(ctx, root: nil)
           find(ctx, root) { |attrs| attrs["role"] == "main" }
         end
 
-        def development(ctx, root: nil)
-          find(ctx, root) { |attrs| attrs["role"] == "development" }
+        def development(ctx, root: nil, create_if_missing: true)
```

Comment:
This method is totally fair. However, personally, in order to avoid the `create_if_missing` parameter, we could have two methods: `development` and `development!` (this second one would create the theme if it's missing).

Still, feel free to keep it as is as well, the code is super clear :)

Reply:
Thanks **you**, @kilgore5! 🚀